sql/sql_alter.h

```diff
@@ -108,6 +108,12 @@ class Alter_info
   // Number of partitions.
   uint                          num_parts;
 private:
+#ifdef WITH_WSREP
+  /* wsrep patch needs to peak the algorithm clause used in ALTER statement
```

Comment:
1. Typo: I believe you meant the word *`pick`, not `peak`, here.
2. Why not simply make the `algorithm` member `public:` _unconditionally_?